### ISSUE TITLE
cmd/gf: gf gen dao 是不是可以加一个参数选项 开启后直接生成通用的增删改查及分页？

### ISSUE THREAD

**Issue by wangle201210** [gogf/gf#3957]
### Description

如题，增加后可以极大提高简单业务的开发效率

**Comment by gqcn** [comment]
@wangle201210 Yes, this is already in the plan: https://wiki.goframe.org/pages/viewpage.action?pageId=30737826 However, this feature is more complex than we initially expected. It requires some time for proper design and implementation.
